src/decimal-outline-pdf.js

```diff
@@ -0,0 +1,53 @@
+const asciidoctor = require("asciidoctor")();
```

Comment:
These two custom converters should be de-duplicated, and the bulk of the "decimal-outline" logic moved to its own single .js file.